### ISSUE TITLE
Blog: Sharing icons location in mobile + tablet

### ISSUE THREAD

**Issue by venturewise** [hlxsites/merative#196]
Our Franklin pages currently have the sharing icons at the end of the article for mobile + tablet breakpoints:
https://main--merative2--hlxsites.hlx.page/blog/4-ways-rsna-laid-out-imaging-ai-future
![image](https://user-images.githubusercontent.com/3883795/231752243-78e5848a-befe-4d77-a356-269a37bf24dd.png)

But Figma mobile + tablet designs have the sharing icons above the "left" sub-navigation area, and below the photo + author byline.

FIgma:
![image](https://user-images.githubusercontent.com/3883795/231752026-659f143c-0b66-457e-b326-0c99498c1991.png)
https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?node-id=9144-42221&t=NUrRjiq91nWQfSOW-0

**Comment by venturewise** [comment]
Question: 
Now that I look at it, I see that we'd have to inject the left nav into the middle of the main content coming from, just like we are doing for author-bio. However, instead of populating from the metadata, can we populate a copy from the blog-left-nav and hide it in desktop (when the scrolling left nav version is currently shown)?

**Comment by amol-anand** [comment]
We could also simply just have the same structure across all breakpoints? Social icons show up under or over left nav in all scenarios?

**Comment by venturewise** [comment]
It doesn't make sense to have the "in this issue" under all the content in tablet/mobile. Nor would it make sense to be above the H1.

**Comment by amol-anand** [comment]
No. this should be automated and not have an author have to add a block called left nav randomly in the middle of their content.

**Comment by venturewise** [comment]
Updating to add the Slack conversation from Keith that this feature is still a WIP on their side.
![image](https://user-images.githubusercontent.com/3883795/233637351-17ba424c-f2c4-47d5-b6d7-395bbb976dc1.png)
